src/bin.ts

```diff
@@ -194,6 +199,24 @@ const deploy = async ({ yes }: { yes: boolean }) => {
         // now we play the waiting game.
         await streamToPromise(stream as any as Readable); // todo: find out why the typing won't allow this as-is
         await Promise.all(promises);
+
+        if (config.removeNonexistentObjects) {
+            let objectsToRemove = objects.map(obj => ({Key: <string>obj.Key})).filter(obj => obj.Key && !isKeyInUse[obj.Key]);
+
+            for (let i = 0; i < objectsToRemove.length; i += OBJECTS_TO_REMOVE_PER_REQUEST) {
+                let objectsToRemoveInThisRequest = objectsToRemove.slice(i, i + OBJECTS_TO_REMOVE_PER_REQUEST);
```

Comment:
Please use `const`'s if you're not mutating your variable

Reply:
Good call, I'm still getting used to the way const is used in JS. I'm used to const only being used for all-caps constants at the top of a file.

Reply:
Yea... as with most concepts that JS "borrows" from other languages, JS consts are very different from other languages.